multisrc/overrides/madara/olaoe/src/Olaoe.kt

```diff
@@ -0,0 +1,22 @@
+package eu.kanade.tachiyomi.extension.ar.olaoe
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.SChapter
+import okhttp3.Request
+import org.jsoup.nodes.Element
+
+class Olaoe : Madara("Olaoe", "https://olaoe.cyou", "ar") {
+    override val mangaSubString = "works"
+    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/page/$page/?m_orderby=views", headers)
+    override fun chapterListSelector() = "li.wp-manga-chapter:not(.premium-block)" // Filter fake chapters
+    override fun chapterFromElement(element: Element): SChapter {
+        val chapter = SChapter.create()
+
+        element.select("a").let {
+            chapter.url = it.attr("href").substringAfter(baseUrl)
```

Comment:
There's a `setUrlWithoutDomain()` function that you can use instead of doing it this way.

Reply:
i will us it